File: android/versioned-abis/expoview-abi39_0_0/src/main/java/abi39_0_0/expo/modules/camera/tasks/FaceDetectorAsyncTaskDelegate.java
package abi39_0_0.expo.modules.camera.tasks;

import android.os.Bundle;

import java.util.List;

import abi39_0_0.org.unimodules.interfaces.facedetector.FaceDetector;

public interface FaceDetectorAsyncTaskDelegate {
  void onFacesDetected(List<Bundle> faces);
  void onFaceDetectionError(FaceDetector faceDetector);
  void onFaceDetectingTaskCompleted();
}

File: android/versioned-abis/expoview-abi47_0_0/src/main/java/abi47_0_0/expo/modules/camera/tasks/FaceDetectorAsyncTaskDelegate.kt
package abi47_0_0.expo.modules.camera.tasks

import android.os.Bundle
import abi47_0_0.expo.modules.interfaces.facedetector.FaceDetectorInterface

interface FaceDetectorAsyncTaskDelegate {
  fun onFacesDetected(faces: List<Bundle>)
  fun onFaceDetectionError(faceDetector: FaceDetectorInterface)
  fun onFaceDetectingTaskCompleted()
}
